src/ucp/core/ucp_am.c

```diff
@@ -899,6 +899,14 @@ ucp_am_send_req(ucp_request_t *req, size_t count,
     return req + 1;
 }
 
+static UCS_F_ALWAYS_INLINE ssize_t
+ucp_am_get_short_max_reply(ucp_ep_h ep)
+{
+    return ucs_min(ucp_ep_config(ep)->am_u.max_eager_short -
+                   sizeof(ucs_ptr_map_key_t),
+                   UCP_AM_SHORT_REPLY_MAX_SIZE);
```

Comment:
you can remove `UCP_AM_SHORT_REPLY_MAX_SIZE` definition now and use `UCS_ALLOCA_MAX_SIZE` instead

Reply:
OK, I removed it